softdevice_controller: Update library selection (#4212)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

.. rst-class:: v1-5-0

> DRGN-15475: Samples using both the advertising and scanning but not connected state will fail to initialize the Bluetooth driver.
>   Samples using both the advertising and scanning but not connected state will fail to initialize the Bluetooth driver.

Reviewer: The first line is the title for the known issue, the second is the description. So the first can be more general, while the second should be more descriptive. For example:
```suggestion
DRGN-15475: Samples might not initialize the Bluetooth driver correctly
  Samples that use both the advertising and the scanning state, but not the connected state, fail to initialize the Bluetooth driver.
  As a result, they report the following error/hang during startup/cannot initialize a connection/...?
```
Also, what do we mean with "Bluetooth driver"? Are we talking about the SoftDevice Controller? Or some driver in Zephyr?

Reviewer 2: "SoftDevice controller HCI driver" Would be most descriptive I think.

>   **Workaround:** Manually add CONFIG_SOFTDEVICE_CONTROLLER_MULTIROLE=y to the project configuration.

Reviewer: ```suggestion
  **Workaround:** Manually enable :option:`CONFIG_SOFTDEVICE_CONTROLLER_MULTIROLE` for the project.
```

.. rst-class:: v1-5-0

DRGN-15475: Samples might not initialize the SoftDevice Controller HCI driver correctly

>   Samples using both the advertising and scanning but not connected state will fail to initialize the SoftDevice Controller HCI driver.
>   As a result, the function :cpp:func:`bt_enable()` will return an error code.

Reviewer: ```suggestion
  Samples using both the advertising and the scanning state, but not the connected state, fail to initialize the SoftDevice Controller HCI driver.
  As a result, the function :c:func:`bt_enable()` returns an error code.
```
